src/lib/fetch.ts

```diff
@@ -67,8 +68,12 @@ async function fetchAccount(
     accountInfo.publicKey instanceof PublicKey
       ? accountInfo.publicKey.toBase58()
       : accountInfo.publicKey;
+  let tokenIdBase58 = 
+    accountInfo.tokenId instanceof Field
```

Comment:
unfortunately we can't use `instanceof Field` (because the Field class is proxied, so this actually won't be true for fields)

but `typeof tokenId === "string"` will do it!